### ISSUE TITLE
Consider switching to the sensu-install tool

### ISSUE THREAD

**Issue by sstarcher** [sstarcher/docker-sensu#51]

**Comment by ppawiggers** [comment]
Simple example for installing plugins with sensu-install: https://github.com/ppawiggers/docker-sensu/blob/master/bin/install

**Comment by sstarcher** [comment]
The current install in `docker-sensu` supports a good bit more options, but ya the conversion should be fairly easy.  I just need to drop some things in the script.

**Comment by ppawiggers** [comment]
Agree, that's why I did not submit a pull request for this feature, I currently just use this for myself.

**Comment by guimaluf** [comment]
Seems like load-checks doesn't work in current plugins install method.
Cause load-checks requires a relative lib that isn't installed in the proper directory with gem build method.

```
/opt/sensu/embedded/bin/metrics-load.rb:37:in `require_relative': cannot load such file -- /opt/sensu/embedded/lib/sensu-plugins-load-checks/load-average.rb (LoadError)
```

**Comment by sstarcher** [comment]
`require_relative` should be fixed in that plugin.  Plugins should not be using `require_relative`

**Comment by sstarcher** [comment]
This is a proper example of including a local library https://github.com/sensu-plugins/sensu-plugins-aws/blob/ffe7b24cf2b3f3f7ee3e92b08d40320377e2baa4/bin/check-beanstalk-elb-metric.rb#L30
